src/services/affinidi/errors.ts

```diff
@@ -65,6 +66,10 @@ export function handleServiceError(
   serviceErrorMessageHandler?: (response: any) => string | null,
 ): never {
   if (error instanceof AxiosError && error.response) {
+    if (error.response.status >= 500) {
+      credentialsVault.clear()
```

Comment:
dumb question:  don't we need to return after the `credentials` are cleared ?

Reply:
After moving to BFF, `sessionId` became the only thing that is stored in file ([see service for the reference](https://github.com/affinidi/affinidi-cli/blob/main/src/services/credentials-vault.ts#L3)).

Reply:
Throwing `CLIError` in the end of the block returns a nice message to the user.